build/common/docker-build-push.yml

```diff
@@ -0,0 +1,63 @@
+# DESCRIPTION:
+# Builds and pushes a docker image for dicom-server and dicom-cast
+
+parameters:
+  - name: tag
+    type: string
+  - name: push
+    type: boolean
+    default: true
+
+steps:
+  - task: Docker@2
+    displayName: Login
+    inputs:
+      command: login
+      containerRegistry: 'dicomoss.azurecr.io'
```

Comment:
Do you want to keep this as variable in this file. So it is easy to update one line

Reply:
Sure, I can move it